tensorboard/webapp/widgets/data_table/column_selector_component.ng.html

```diff
@@ -0,0 +1,39 @@
+<!--
+@license
+Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div class="contents">
+  <mat-form-field class="search-area">
+    <mat-icon matPrefix class="search-icon" svgIcon="search_24px"></mat-icon>
+    <input
+      matInput
+      #search
+      [(ngModel)]="searchInput"
```

Comment:
This does seem like an appropriate use case for 2 way binding, but I want to point out that I'm using it because it is a controversial subject.